### ISSUE TITLE
Add 'adding species' preprint to `docs/introduction.rst` for 0.2.0

### ISSUE THREAD

**Issue by nspope** [popsim-consortium/stdpopsim#1394]
As the 0.2.0 release is accompanying the preprint, it'd be good to have a citation/link for this in the stable docs, alongside Adrion et al.

**Comment by petrelharp** [comment]
Agree. I guess this means we should preprint first, but be ready to release at a moment's notice?

**Comment by nspope** [comment]
FYI @igronau-- I think we'll synchronize the release with the preprint after all, so that there's a reference+DOI in the stable docs.
